pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -86,9 +96,16 @@ func LoadClientCert(kubeconfigPath string, bootstrapPath string, certDir string,
 	if err != nil {
 		return err
 	}
-	if err := certutil.WriteCert(certPath, certData); err != nil {
+	if err = certutil.WriteCert(certPath, certData); err != nil {
 		return err
 	}
+	cert, err := x509.ParseCertificate(certData)
+	if err != nil {
+		return err
+	}
+	if cert.NotAfter.Sub(time.Now()) <= time.Duration(0) {
```

Comment:
I don't think this belongs here... there is client side code to attempt to renew the certificate when expired, but client side timestamp validation doesn't seem right